scripts/linkml_to_flat_synapse_jsonschema.py

```diff
@@ -0,0 +1,121 @@
+import subprocess
+import sys
+import json
+from pathlib import Path
+import jsonref
+
+def run_gen_json_schema(linkml_yaml, class_name, tmp_json):
```

Comment:
Use type linking

```
def run_gen_json_schema(linkml_yaml: str, class_name: str, tmp_json: str)
```

Add docstrings and add Argument definitions.  So an example is this:


```
def get_assay_dataframe(filepath_list: list) -> pd.DataFrame:
    """Reads in assay_information.yaml file
        and outputs it as a dataframe

    Args:
        filepath_list (list): list of files

    Raises:
        ValueError: thrown if read error with file

    Returns:
        pd.DataFrame: dataframe version of assay info file
    """
```